rocks-lib/src/build/mod.rs

```diff
@@ -94,10 +95,14 @@ pub async fn build(progress: &MultiProgress, rockspec: Rockspec, config: &Config
     // operations in the temporary directory itself and then copy all results over once they've
     // succeeded.
 
-    let output_paths = tree.rock(&LuaPackage::new(
-        rockspec.package.clone(),
-        rockspec.version.clone(),
-    ))?;
+    // TODO(vhyrro): The `build` operation needs more metadata, specifically the dependency
+    // and pinning information.
+    let package = LuaPackage::from(
+        &RemotePackage::new(rockspec.package.clone(), rockspec.version.clone()),
+        LockConstraint::Unconstrained,
+    );
```

Comment:
Since this is quite serious I'm in the process of making a follow-up pull request to make this work.

Reply:
#121